bounties/packagist/opencart/opencart/2/vulnerability.json

```diff
@@ -0,0 +1,51 @@
+{
+  "PackageVulnerabilityID": "2",
+  "DisclosureDate": "9-12-2020",
+  "AffectedVersionRange": "",
```

Comment:
Please define the `versions` affected field :smile:

Reply:
done